test-packages/macro-tests/vendor/prepend/order.js

```diff
@@ -0,0 +1 @@
+self.ORDER = [];
```

Comment:
this needed to be moved out of `index.html` as that script tags in index.html are not loaded in fastboot mode.